packages/perseus/src/renderer-util.test.ts

```diff
@@ -39,7 +105,7 @@ describe("emptyWidgetsFunctional", () => {
     it("properly identifies empty widgets", () => {
         // Arrange
         const widgets: PerseusWidgetsMap = {
-            "dropdown 1": testDropdownWidget,
+            "dropdown 1": getTestDropdownWidget(),
```

Comment:
I used a function for the new tests, so I changed the old one to be consistent with the new one.

IMO it's better to use a function because it ensures tests are as isolated as possible. Using the same object for multiple tests means one test can mutate the object that is then passed to another test.